src/components/Client.ts

```diff
@@ -56,9 +56,19 @@ async function getBuckets () {
   return []
 }
 
+async function getMeasurements (bucket: string) {
+  if (Status.Current) {
+    const measurements = await Queries.measurements(Status.Current, bucket)
+    return (measurements?.rows || []).map(row => row[0])
+  }
+
+  return []
+}
+
 const createStream = () => {
   const server = new Server(true, false)
```

Comment:
curios, what is the two bool args for?

Reply:
https://github.com/influxdata/flux-lsp/blob/master/src/wasm/mod.rs#L46